tools/assimp_view/stdafx.h

```diff
@@ -23,7 +23,6 @@
 #define _WIN32_IE 0x0600    // �ndern Sie dies in den geeigneten Wert f�r andere Versionen von IE.
 #endif
 
-#define WIN32_LEAN_AND_MEAN     // Selten verwendete Teile der Windows-Header nicht einbinden.
```

Comment:
Is this because of a "duplicate definition" warning? If so you should prefer


```
#ifndef WIN32_LEAN_AND_MEAN
#define WIN32_LEAN_AND_MEAN
#endif  // WIN32_LEAN_AND_MEAN
```


It includes less stuff and speeds up compile times.

Reply:
WIN32_LEAN_AND_MEAN is defined project-wide already. It appears in the root CMakeLists.txt.
I felt any explicit usage of it in code was redundant.